### PR TITLE
Add a case which allows Windows virtio ISO to be a block device

### PR DESCRIPTION
**Pull request by xiaodwan**
bz#1727164

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.virtio_iso_blk.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (660.49 s)

**Comment by xiaodwan** [comment]
rhel8:
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.virtio_iso_blk.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1084.33 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh @yantian0 Please help to review this PR, thanks.

File: v2v/tests/src/function_test_esx.py [hunk 1]
```diff
@@ -612,6 +624,9 @@ def check_result(result, status_error):
                     params['tmp_mount_point'],
                     'iso9660')
             os.environ.pop('VIRTIO_WIN')
+        if checkpoint == 'virtio_iso_blk':
+            process.run('losetup -d %s' % free_loop_dev, shell=True)
```
Comment (chloerh) [inline v2v/tests/src/function_test_esx.py:628]: Do we need to print out command output? If we do, we could set 'verbose=True'. Same for line 548 and 551. Better for future debug.

Comment (xiaodwan) [reply v2v/tests/src/function_test_esx.py:628]: Yes, verbose is true by default in process.run.

**Pull request merged**